src/pages/CreateAccount/CreateAccount.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { NavLink, Navigate } from "react-router-dom";
+import { ToastContainer, toast } from "react-toastify";
+import { v4 as uuidv4 } from "uuid";
+
+import { isObject } from "./helper";
+
+import { createAccountUser } from "../../redux/store/usersReducer";
+import Tittle from "../../Komponents/TittleColection/Tittle";
+
+import "react-toastify/dist/ReactToastify.css";
+
+import style from "../CreateAccount/CreateAcc.module.scss";
+
+const CreateAccount = () => {
+  const dispatch = useDispatch();
+  const [dataToLogIn, setdataToLogIn] = useState({});
+  const [canLogIn, setCanLogIn] = useState(false);
+  const [activeInp, setActiveInp] = useState(true);
+  const [counter, setCounter] = useState(0);
+
+  const showMessageError = () => {
```

Comment:
a lot of repeatable toast calls. Create that toast component in your Component folder and reuse it. U should create a few components for success, error, e.t.c. That component shoud get message from props and ohter unnecessary props for modification